go.mod

```diff
@@ -2,4 +2,7 @@ module k8s.io/klog/v2
 
 go 1.12
 
-require github.com/go-logr/logr v0.1.0
+require (
+	github.com/go-logr/logr v0.1.0
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
```

Comment:
Rather than re-adding glog back to `go.mod` should we instead create a separate `go.mod` file for the example code that is pulling `glog` in?

Reply:
good idea .